src/platform/forms-system/src/js/widgets/TextWidget.jsx

```diff
@@ -10,8 +10,8 @@ export default function TextWidget(props) {
   }
 
   const inputProps = {
-    ...(props.minLength && { min: props.minLength }),
-    ...(props.maxLength && { max: props.maxLength }),
+    ...(props.schema.minValue && { min: props.schema.minValue }),
+    ...(props.schema.maxValue && { max: props.schema.maxValue }),
```

Comment:
@cvalarida I was accessing the wrong property - this should work now 😬

Reply:
:man_facepalming: